src/metabase/api/session.clj

```diff
@@ -232,9 +232,17 @@
   (forgot-password-impl email)
   api/generic-204-no-content)
 
-(def ^:private ^:const reset-token-ttl-ms
-  "Number of milliseconds a password reset is considered valid."
-  (* 48 60 60 1000)) ; token considered valid for 48 hours
+
+(defsetting reset-token-ttl-hours
+            (deferred-tru "Number of hours a password reset is considered valid.")
+            :visibility :internal
+            :type       :integer
+            :default    48)
```

Comment:
This should only be indented by two spaces

Reply:
sorry bout that, fixed